### DISCUSSION TITLE
multilingual fallbacks for repository README.md files

### DISCUSSION THREAD

**Discussion by emyoulation** [community/community#52292] [Repositories]
### Select Topic Area

Product Feedback

### Body

The feature which displays the README.md automatically when browsing a repository is very useful.  

I just encountered a repository where the owner's native language was Esperanto... so that was the language of their README.md documentation file. The owner did not want to break backwards compatibility for Esperanto readers when adding an english README.md file.

We wanted to add a README.en.md  (or README.en-us.md ) file for the repository's english speakers.  This raised the question of if GitHub would fallback to a README.en.md (or README.en-US.md or README.en-GB.md dialect) file if a README.md did not exist. 

It cannot not fallback in that manner.

The documentation https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes does not include filenaming conventions (neither for case sensitivity nor internationalization/i18n/localization/L10n)

We are considering serving documentation automatically in our open source project's plugins.  It would be convenient to follow the same guidelines as GitHub since that is where the plugins got through beta testing. So we need better defined guidelines.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @emyoulation ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
